src/plugins/intel_cpu/src/emitters/plugin/aarch64/jit_eltwise_emitters.cpp

```diff
@@ -84,28 +89,22 @@ jit_add_emitter::jit_add_emitter(dnnl::impl::cpu::aarch64::jit_generator* host,
 
 size_t jit_add_emitter::get_inputs_count() const { return 2; }
 
-void jit_add_emitter::emit_impl(const std::vector<size_t> &in_vec_idxs, const std::vector<size_t> &out_vec_idxs) const {
-    if (host_isa_ == dnnl::impl::cpu::aarch64::asimd) {
-        emit_isa<dnnl::impl::cpu::aarch64::asimd>(in_vec_idxs, out_vec_idxs);
-    } else {
-        OV_CPU_JIT_EMITTER_THROW("Can't create jit eltwise kernel");
-    }
-}
-
-template <dnnl::impl::cpu::aarch64::cpu_isa_t isa>
+template <dnnl::impl::cpu::aarch64::cpu_isa_t isa, typename type>
 void jit_add_emitter::emit_isa(const std::vector<size_t> &in_vec_idxs, const std::vector<size_t> &out_vec_idxs) const {
-    OV_CPU_JIT_EMITTER_ASSERT(exec_prc_ == ov::element::f32, "unsupported precision: " + exec_prc_.to_string());
+    OV_CPU_JIT_EMITTER_ASSERT(
+            (exec_prc_ == ov::element::f16) || (exec_prc_ == ov::element::f32),
```

Comment:
Please use `one_of` instead

Reply:
thanks for comment, not neccessary: more general comment is used